app/views/transaction_entries/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<section class="container mt-4">
+  <h2 class="text-center">DETAILS</h2>
+  <div class="mt-4">
+    <h1><%= @category.name %> Transactions</h1>
+    <p>Total Amount: $<%= @total_sum %></p>
+
+    <% @transaction_entries.each do |transaction| %>
+      <div class="card mb-3">
+        <div class="card-body">
+          <h3 class="card-title"><%= transaction.name %></h3>
+          <p class="card-text">Amount: $<%= transaction.amount %></p>
+        </div>
+      </div>
+    <% end %>
+  </div>
+
+  <div class="d-grid gap-2 mt-4">
+    <%= link_to 'Add a New Transaction', new_category_transaction_entry_path(@category), class: 'btn btn-outline-dark' %>
+    <%= link_to 'Back', categories_path, class: 'btn btn-outline-primary' %>
+  </div>
+</section>
```

Comment:
- [ ] Since this is your Details page, I will suggest you style the navbar to be similar to that of the given design. So it has the `Details` header and back button in the navbar :+1: 
- [ ] So you can remove the  back button you have in your design currently and style the `Add transaction` button to look similar to the one in my earlier comment :+1: 

| Your Design | The Given Design |
|--------|--------|
| ![image](https://github.com/MMhaidari/rails-solo-capstone/assets/102699958/3a6f4037-a91f-4009-8c01-9e4f30eb4581) | ![image](https://github.com/MMhaidari/rails-solo-capstone/assets/102699958/e3f99d61-b834-4763-80e7-9423864ea422)| 

- [ ] You can remove this part of your Details page and leave just the `Total Amount` under the `navbar` after which you also need to style the added transactions as seen in the original design :wink: 
![image](https://github.com/MMhaidari/rails-solo-capstone/assets/102699958/63573ffb-0ad0-43a2-95ca-4acb7885434f)

- [ ] You can style the `Total Amount` similar to  `Gift Credit `as seen in the image below. Directly under your Details `Navbar` after which the added transactions can be styled as seen below and the `Add Transaction` button replacing `Scan` :wink: 
- Given Design
![image](https://github.com/MMhaidari/rails-solo-capstone/assets/102699958/6cf6c2d6-ae8a-4038-9797-23265c3fc217)